parser-typechecker/src/Unison/Runtime/MCode.hs

```diff
@@ -899,12 +901,6 @@ emitFunction _ grpn rec ctx (FVar v) as
       App False (Env grpn j) as
   | otherwise = emitSectionVErr v
 emitFunction rns _ _ _ (FComb r) as
-  | False -- known saturated call
```

Comment:
These seem like placeholders for unimplemented faster-path things that we shouldn't necessarily delete

Reply:
See an earlier resolved comment from Dan where he says it's fine to delete.

Ironically, I also tried commenting out, but `-- |` fails to parse here haha.

Regardless, it's removed in the commit such that it will be revived (as a comment) if we upgrade to 9.2.3 by reverting https://github.com/unisonweb/unison/pull/3304/commits/2de0751669606be3bee5a12542d8ea39d3633ca1